Sources/Sentry/include/SentrySDKInternal.h

```diff
@@ -282,7 +282,8 @@ SENTRY_NO_INIT
 
 #if TARGET_OS_IOS && SENTRY_HAS_UIKIT
 
-@property (nonatomic, class, readonly) SentryFeedbackAPI *feedback;
+@property (nonatomic, class, readonly) SentryFeedbackAPI *feedback NS_EXTENSION_UNAVAILABLE(
```

Comment:
`m`: Why is this marked unavailable? The `SentryFeedbackAPI`?

Reply:
I just noticed my feedback got cut off. My follow-up would have been that `SentryFeedbackAPI` will be extended with additional options down the road, so maybe we should make it accessible so we are not blocking ourselves.